client/src/hooks/datafiles/mutations/useCompress.ts

```diff
@@ -78,18 +78,22 @@ function useCompress() {
     }
 
     if (!allocationForCompress) {
```

Comment:
!allocationForCompress currently evaluates as false for TACC-ACI, but then fails in slurm due to no existing allocation in Frontera.